inst/scripts/make-blueprint_encode-data.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Obtaining reference data from Blueprint and ENCODE"
+author: "Friederike Duendar"
+date: "8/9/2019"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+library(BiocStyle)
+knitr::opts_chunk$set(error=FALSE, message=FALSE, warning=FALSE)
+```
+
+Dvir Aran's original `SingleR` github repository contains Robjects with normalized expression values from reference data sets such as those collected by Blueprint & Encode, the Human Primary Cell Atlas, and ImmGen.
```

Comment:
If you use `r Githubpkg("dviraran/SingleR")`, you get a link for free.

Reply:
nice